includes/Loading/LoadCombination.js

```diff
@@ -1,19 +1,259 @@
+        enabled = true;
+    }
+    this.load_combination.consider_creep_caused_by_permanent_loading = enabled;
+    if (enabled) {
+        if (typeof creep_caused_by_permanent_loading_case === "undefined") {
+            ASSERT("Loading case must be defined");
+        }
+        this.load_combination.creep_caused_by_permanent_loading_case = creep_caused_by_permanent_loading_case;
+    }
+    return this.load_combination;
+}
+
+
+/**
+ * Consider construction stage
+ * @param {Object}  construction_stage  Construction stage, can be undefined (in case of disabling)
+ * @param {Boolean} enabled             Enable/disable construction stage, can be undefined (true as default)
+ * @returns Modified load combination
+ */
+ LoadCombination.prototype.ConsiderConstructionStage = function (construction_stage,
+    enabled) {
+    if (typeof enabled === "undefined") {
+        enabled = true;
+    }
+    this.load_combination.consider_construction_stage = enabled;
+    if (enabled) {
+        if (typeof construction_stage === "undefined") {
+            ASSERT("Construction stage must be defined");
+        }
+        this.load_combination.construction_stage = construction_stage;
+    }
+    return this.load_combination;
 }
+
+
+/**
+ * Creates load combination (private)
+ * @param {Number}  no      Load combination index, can be undefined
+ * @param {String}  name    Name, can be undefined
+ * @param {String}  comment Comment, can be undefined
+ * @param {Object}  paramns Additonal parameters, can be undefined
+ * @returns Created load combination
+ */
+function craeteBaseLoadCombination (no,
+    name,
+    comment,
+    params) {
+    if (typeof no === "undefined") {
+        no = load_combinations.count() + 1;
+    }
+    var load_combination = load_combinations.create(no);
+    if (typeof name !== "undefined") {
+        load_combination.name = name;
+    }
+    load_combination
+    set_comment_and_parameters(load_combination, comment, params)
+    return load_combination;
+};
+
+
+/**
+ * Gets all available analysis types strings
+ */
+function get_analysis_types () {
+    console.log(Object.keys(analysis_types));
+};
+
+
+/**
+ * Gets all available initial state definition types strings
+ */
+ function get_initial_state_definition_types () {
+    console.log(Object.keys(initial_state_definition_types));
+};
+
+const analysis_types = {
+    "ANALYSIS_TYPE_STATIC" : "Static Analysis",
+    "ANALYSIS_TYPE_STATIC_TIME_DEPENDENCE" : "Static Analysis | Time-Dependent Analysis (TDA)",
+    "ANALYSIS_TYPE_STATIC_CREEP_AND_SHRINKAGE" : "Static Analysis | Creep & Shrinkage (Linear)",
+    "ANALYSIS_TYPE_HARMONIC_RESPONSE_ANALYSIS" : "Harmonic Response Analysis"
+};
+
+const initial_state_definition_types = {
+    "DEFINITION_TYPE_FINAL_STATE" : "Final State",
+    "DEFINITION_TYPE_STRAINS" : "Strains",
+    "DEFINITION_TYPE_STIFFNESS" : "Stiffness",
+    "DEFINITION_TYPE_STRAINS_WITH_USER_DEFINED_FACTORS" : "Strains with user-defined factors"
+};
```

Comment:
I'm missing function for assigment of load cases to load combination
there is property items - nested table which contains assingment of load cases with multipliers. Every row in items represent one load case and its factor